### PR TITLE
Add new committer

### PR DESCRIPTION
**Pull request by otterc**
Adding a new committer

### CODE REVIEW

**Comment by otterc** [comment]
@pan3793 PTAL

**Comment by waitinfuture** [comment]
Congrats!

**Comment by cfmcgrady** [comment]
Congrats!

File: docs/community/committer.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ license: |
 | Yihe Li         |
 | Wei Wu          |
 | Zhongqiang Chen |
-
+| Chandni Singh   |
```
Comment (cfmcgrady) [inline docs/community/committer.md]: nit: sort alphabetically

**Review by pan3793 — APPROVED** [review]

**Comment by mridulm** [comment]
Congratulations @otterc !

**Comment by AngersZhuuuu** [comment]
Congrats！

**Comment by SteNicholas** [comment]
@otterc Congrats!

**Pull request merged**